model/cicd/spans.yaml

```diff
@@ -0,0 +1,26 @@
+groups:
+  - id: span.cicd.pipeline.run.server
+    type: span
+    span_kind: server
+    brief: 'Semantic Convention for pipeline runs'
+    stability: development
+    attributes:
+      - ref: cicd.pipeline.result
+        requirement_level: required
+  - id: span.cicd.pipeline.task.internal
+    type: span
+    span_kind: internal
+    brief: 'Semantic Convention for tasks in a pipeline run'
```

Comment:
```suggestion
    brief: This span describes CICD pipeline task execution
```

Reply:
Done in 637ee78fcca2927a7969eb5abacc8de0de1d690f.